### PR TITLE
Fix npe issue when formatting the kyuubi-ctl output

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

Fix npe issue:

```
Exception in thread "main" java.lang.NullPointerException
	at com.jakewharton.fliptables.FlipTable.<init>(FlipTable.java:42)
	at com.jakewharton.fliptables.FlipTable.of(FlipTable.java:20)
	at org.apache.kyuubi.ctl.util.Tabulator$.formatTextTable(Tabulator.scala:37)
	at org.apache.kyuubi.ctl.util.Tabulator$.format(Tabulator.scala:25)
	at org.apache.kyuubi.ctl.util.Render$.renderBatchListInfo(Render.scala:39)
	at org.apache.kyuubi.ctl.cmd.list.ListBatchCommand.$anonfun$render$1(ListBatchCommand.scala:58)
	at org.apache.kyuubi.ctl.cmd.Command.info(Command.scala:86)
	at org.apache.kyuubi.ctl.cmd.list.ListBatchCommand.render(ListBatchCommand.scala:58)
	at org.apache.kyuubi.ctl.cmd.list.ListBatchCommand.render(ListBatchCommand.scala:26)
	at org.apache.kyuubi.ctl.cmd.Command.$anonfun$run$1(Command.scala:47)
	at org.apache.kyuubi.ctl.cmd.Command.$anonfun$run$1$adapted(Command.scala:47)
	at scala.Option.foreach(Option.scala:407)
	at org.apache.kyuubi.ctl.cmd.Command.run(Command.scala:47)
	at org.apache.kyuubi.ctl.ControlCli.doAction(ControlCli.scala:45)
	at org.apache.kyuubi.ctl.ControlCli$$anon$1.doAction(ControlCli.scala:78)
	at org.apache.kyuubi.ctl.ControlCli$.main(ControlCli.scala:86)
	at org.apache.kyuubi.ctl.ControlCli.main(ControlCli.scala)
```

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/util/TabulatorSuite.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    val expected =
+      """
+        |╔════╤════╗
+        |║ c1 │ c2 ║
+        |╠════╪════╣
+        |║ 1  │    ║
+        |╟────┼────╢
+        |║    │ 2  ║
+        |╚════╧════╝
+        |""".stripMargin
```
Comment (ulysses-you) [inline kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/util/TabulatorSuite.scala:35]: It seems a bug of tabulator. The null and empty string should not be regarded as equivalent.

is this better like this ?
```
val rows: Array[Array[String]] = Array(Array("1", ""), Array(null, "2"))

|╔═════╤════╗
|║  c1 │ c2 ║
|╠═════╪════╣
|║  1  │    ║
|╟─────┼────╢
|║ null│ 2  ║
|╚═════╧════╝
```

Comment (ulysses-you) [inline kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/util/TabulatorSuite.scala:35]: It seems a bug of tabulator. The null and empty string should not be regarded as equivalent.

is this better like this ?
```
val rows: Array[Array[String]] = Array(Array("1", ""), Array(null, "2"))

|╔═════╤════╗
|║  c1 │ c2 ║
|╠═════╪════╣
|║  1  │    ║
|╟─────┼────╢
|║ null│ 2  ║
|╚═════╧════╝
```

Comment (turboFei) [reply kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/util/TabulatorSuite.scala]: thanks,

how about `N/A`

**Review by yaooqinn — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**